README.md

```diff
@@ -1,1329 +1,100 @@
-used as a platform to combine existing data-science components into a new
-powerful application that can be deployed remotely for many users. Some of the
-higher level components that can be used are text editors, terminals,
-notebooks, interactive widgets, filebrowser, renderers for different file
-formats that provide access to an enormous ecosystem of libraries from
-different languages.
-
-## Prerequesites ##
-
-Writing an extension is not particularly difficult but requires very basic
-knowledge of javascript and typescript and potentially Python.
-
-_Don't be scared of typescript, I never coded in typescript before I touched 
-JupyterLab but found it easier to understand than pure javascript if you have a 
-basic understanding of object oriented programming and types._
-
-## 1 Hello World: Setting up the development environment ##
-
-#### The template folder structure ####
-
-Writing a JupyterLab extension usually starts from a configurable template. It
-can be downloded with the `cookiecutter` tool and the following command:
-
-```bash
-cookiecutter https://github.com/JupyterLab/extension-cookiecutter-ts
-```
-
-`cookiecutter` asks for some basic information that could for example be setup
-like this:
-```bash
-author_name []: tuto
-extension_name [JupyterLab_myextension]: 1_hello_world
-project_short_description [A JupyterLab extension.]: minimal lab example
-repository [https://github.com/my_name/JupyterLab_myextension]: 
-```
-
-The cookiecutter creates the directory `1_hello_world` [or your extension name]
-that looks like this:
+The goal of this repository is to show how to develop extensions on [JupyterLab](https://github.com/jupyterlab/jupyterlab). 
+It is presented as short tutorial series.
 
 ```bash
-1_hello_world/
-
-├── README.md
-├── package.json
-├── tsconfig.json
-├── src
-│   └── index.ts
-└── style
-    └── index.css
+git clone https://github.com/jtpio/jupyterlab-extension-tutorial.git && 
+  cd jupyterlab-extension-tutorial && \
+  conda env create -f environment.yml && \
+  conda activate jupyterlab-extension-tutorial && \
+  cd hello_world && \
+  npm install && \
+  npm run build && \
+  jupyter labextension link .
+# In another terminal
+jupyter lab --watch
+# open http:/localhost:8888?token=???
 ```
 
-* `README.md` contains some instructions
-* `package.json` contains information about the extension such as dependencies
-* `tsconfig.json` contains information for the typescript compilation
-* `src/index.ts` _this contains the actual code of our extension_
-* `style/index.css` contains style elements that we can use
+## Develop by Examples
 
-What does this extension do? You don't need a PhD in Computer Science to take a
-guess from the title of this section, but let's have a closer look:
+You may find easier to learn how to extend `by examples` instead of going through the [RTFM](https://en.wikipedia.org/wiki/RTFM). Start with the `hello_world` and jump then to topic you are interested into.
```

Comment:
What do you think of "going through the documentation", instead of "going through the RTFM"?